playback/src/audio_backend/pulseaudio.rs

```diff
@@ -12,39 +12,36 @@ const STREAM_NAME: &str = "Spotify endpoint";
 
 pub struct PulseAudioSink {
     s: Option<Simple>,
-    ss: pulse::sample::Spec,
     device: Option<String>,
     format: AudioFormat,
+    pulse_format: pulse::sample::Format,
 }
 
 impl Open for PulseAudioSink {
     fn open(device: Option<String>, format: AudioFormat) -> Self {
-        info!("Using PulseAudio sink with format: {:?}", format);
+        let mut librespot_format = format;
```

Comment:
Maybe use something that states the intention more clearly, like `actual_format`.

Reply:
No problem I can do that.

Reply:
Done